example/app.js

```diff
@@ -33,31 +33,31 @@ ds.discoverForeignKeys('INVENTORY',  show);
 ds.discoverExportedForeignKeys('PRODUCT',  show);
 */
 
-
 var table = (process.argv.length > 2) ? process.argv[2] : 'INVENTORY_VIEW';
 
 ds.discoverSchema(table, {owner: 'STRONGLOOP'}, function(err, schema) {
-    console.log(JSON.stringify(schema));
-    var model = ds.define(schema.name, schema.properties, schema.options);
+  console.log(JSON.stringify(schema));
+  var model = ds.define(schema.name, schema.properties, schema.options);
     // console.log(model);
-    model.all(show);
+  model.all(show);
 });
 
-ds.discoverAndBuildModels('INVENTORY', {owner: 'STRONGLOOP', visited: {}, associations: true}, function (err, models) {
-
-    for(var m in models) {
-        models[m].all(show);
-    };
-
-    models.Inventory.findOne({}, function(err, inv) {
-       console.log(g.f("\nInventory: %s", inv));
-       inv.product(function(err, prod) {
-           console.log(g.f("\nProduct: %s", prod));
-           console.log("\n ------------- ");
+ds.discoverAndBuildModels('INVENTORY',
+{owner: 'STRONGLOOP',
+visited: {},
+associations: true},
+function(err, models) {
+  for (var m in models) {
+    models[m].all(show);
+  };
+
+  models.Inventory.findOne({}, function(err, inv) {
+    console.log(g.f('\nInventory: %s', inv));
+    inv.product(function(err, prod) {
+      console.log(g.f('\nProduct: %s', prod));
+      console.log('\n ------------- ');
            // ds.disconnect(); // This will crash node-oracle as the connection is disconnected while other statements are still running
-       });
     });
+  });
 });
```

Comment:
Your force push overrode the line. Its the ? and the indenting the next lines:

```
a ?
  b :
  c;
```

thing we talked about over Slack.